test/e2e/serial/config/fixture.go

```diff
@@ -77,7 +77,7 @@ func NewFixtureWithOptions(nsName string, options e2efixture.Options) (*E2EConfi
 	if err != nil {
 		return nil, err
 	}
-
+	cfg.Fixture.InitialNRTList = cfg.NRTList
```

Comment:
Is this needed? the test fixture we use here is for infra (numacell ds) so it seems this can be avoided?

Reply:
right, I might have mixed between the setups. Thanks for raising that.